js/hotgraphicView.js

```diff
@@ -53,8 +53,13 @@ define([
 
       const model = this.prepareNarrativeModel();
       const newNarrative = new NarrativeView({ model });
+      // NOTE: if this component is doing its inital render in 'narrative mode',
+      // this.$el.parents() won't exist at this point - which is why the following is
+      // written the way it is, instead of (what would appear to be) the more efficient
+      // this.$el.parents('.component__container')
+      const $container = $('.' + this.model.get('_parentId')).find('.component__container');
```

Comment:
Suggest a more fool-proof way of selecting the element using the data attribute:
```js
$(`[data-adapt-id='${model.get('_parentId')}']`).find('.component__container')
```
or even better using the built-in API:
```js
Adapt.findViewByModelId(model.get('_parentId')).$el.find('.component__container')
```

Reply:
nice one @tomgreenfield I’ll do the latter version tomorrow

Reply:
nice one @tomgreenfield I’ll do the latter version tomorrow

Reply:
Ahh, that'd explain why the syntax was so messy. Apologies for helping cause this one.

Reply:
no problem. i figured the fact that we both missed this meant it would benefit from the comment, to prevent others from falling into the same trap!